web/src/assets/customer/delivery.svg

```diff
@@ -0,0 +1,53 @@
+<svg width="144" height="107" viewBox="0 0 144 107" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please update the PR description. The picture under "After filling in details (confirm button is enabled)" is same as the "confirm button not enabled".

Reply:
@xlai20 Omg that's so weirddd, I remember updating it yesterday when I spotted it too! 😱 😨  Not sure why it's still the old one, maybe I forgot to click the 'Update' button HAHA